src/app/components/collection/collection.component.scss

```diff
@@ -0,0 +1,73 @@
+@use "/src/imports" as *;
```

Comment:
please wrap your targets for better scss application

Reply:
Done !